trpcultivate_germplasm/src/Plugin/TripalImporter/GermplasmCollectionImporter.php

```diff
@@ -0,0 +1,1159 @@
+  upload_title: new TranslatableMarkup('<strong>Population Individuals*</strong>'),
+  use_analysis: FALSE,
+  require_analysis: FALSE,
+  use_button: TRUE,
+  submit_disabled: FALSE,
+  button_text: new TranslatableMarkup('Import'),
+  file_upload: TRUE,
+  file_local: FALSE,
+  file_remote: FALSE,
+  file_required: TRUE,
+  cardinality: 1,
+  menu_path: '',
+  callback: '',
+  callback_module: '',
+  callback_path: '',
+)]
+class GermplasmCollectionImporter extends ChadoImporterBase implements ContainerFactoryPluginInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * The key to reference the validation result array in Drupal storage system.
+   *
+   * @var string
+   */
+  private const VALIDATION_RESULT = 'validation_result';
+
+  /**
+   * The Drupal Messenger Service.
+   *
+   * @var \Drupal\Core\Messenger\MessengerInterface
+   */
+  protected $service_Messenger;
+
+  /**
+   * The Entity Type Manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  protected EntityTypeManager $service_entityTypeManager;
+
+  /**
+   * The TripalCultivate validator plugin manager.
+   *
+   * @var \Drupal\trpcultivate\TripalCultivateValidator\TripalCultivateValidatorManager
+   */
+  protected TripalCultivateValidatorManager $service_validatorPluginManager;
+
+  /**
+   * The TripalCultivate File Template Service.
+   *
+   * @var \Drupal\trpcultivate\Service\TripalCultivateFileTemplateService
+   */
+  protected TripalCultivateFileTemplateService $service_FileTemplate;
+
+  /**
+   * The Drupal Renderer.
+   *
+   * @var \Drupal\Core\Render\Renderer
+   */
+  protected Renderer $service_Renderer;
+
+  /**
+   * A Database query interface for querying Chado using Tripal DBX.
+   *
+   * @var \Drupal\tripal_chado\Database\ChadoConnection
+   */
+  protected ChadoConnection $chado_connection;
+
+  /**
+   * Headers required by this importer.
+   *
+   * @var array
+   *
+   * The following keys are required:
+   * - 'name': The column header name as it should appear in the input file.
+   * - 'description': A user-friendly description of the header that will be
+   *   displayed to the user through the form.
+   * - 'type': one of "required" or "optional" to indicate whether the column
+   *   needs to have values present or not.
+   *
+   * NOTE: Order MUST reflect the desired order of headers in the input file.
+   */
+  protected array $headers = [
+    [
+      'name' => 'Name',
+      'description' => 'The name of the germplasm individual.',
+      'type' => 'required',
+    ],
+    [
+      'name' => 'Type',
+      'description' => 'The type of the germplasm individual, as a controlled vocabulary term.',
+      'type' => 'required',
+    ],
+    [
+      'name' => 'Scientific Name',
+      'description' => 'The scientific name of the germplasm individual.',
+      'type' => 'required',
+    ],
+    [
+      'name' => 'Uniquename',
+      'description' => '(optional) A unique identifier for the germplasm individual.',
```

Comment:
```suggestion
      'description' => 'A unique identifier for the germplasm individual.',
```
'(optional)' is not longer needed